wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsButtonsView.swift

```diff
@@ -72,14 +72,14 @@ struct DeviceDetailsButtonsView: View {
         if let status = viewModel.e2eIdentityCertificate?.status {
             switch status {
             case .valid:
-                if let isExpiring = viewModel.isCertificateExpiringSoon, isExpiring {
+                if viewModel.isSelfClient, viewModel.isCertificateExpiringSoon == true {
```

Comment:
```suggestion
                if viewModel.isSelfClient, viewModel.isCertificateExpiringSoon {
```

Reply:
isCertificateExpiringSoon is optional. Hence, we cannot do that